docs/deployments/debugging-harperdb.md

```diff
@@ -0,0 +1,44 @@
+# Debugging HarperDB
+
+If your HarperDB servers are experiencing performance issues, memory leaks, or errors, enabling debugging can help diagnose the problem. To start debugging, ensure that the HarperDB configuration has debugging enabled.
+
+To learn more, please visit: [HarperDB Configuration - Threads](https://docs.harperdb.io/docs/deployments/configuration#threads)
+
+In most deployments, debugging is already enabled. If it's set up, you can connect to HarperDB using Node DevTools, available via `chrome://inspect` in Chrome (click "Open dedicated DevTools for Node").
+
+## Port Configuration for Debugging
+
+Since DevTools can only connect to one thread per port, we assign specific ports for debugging:
+
+- Main thread: Port 9229
```

Comment:
https://docs.harperdb.io/docs/deployments/configuration#threads
This shows debug.port as 9249? Also it mentions startingPort, which is not in the sample config. Not sure if that's directly relevant to this, but seems like it.

Reply:
We should update those docs to be 9229, that is much more standard. But this should be clarified that this is connects to the ports configured in the starting port range.